lib/user.js

```diff
@@ -257,6 +253,7 @@ class User {
         this._sendMessage(this.bfcpUser.floorRequestStatusMessage(this.lastFloorRelease,
          this.wantedFloorId, RequestStatusValue.Released));
          this.lastFloorRelease = null;
+         delete this.usedServerPorts[user.serverPort];
```

Comment:
user doesn't exist in this context. This should be removed, since this is already done in index.js

Reply:
Updated